metadata.rb

```diff
@@ -10,6 +10,8 @@
   respond_to?(:source_url)
 issues_url 'https://github.com/chef-partners/dcos-cookbook/issues' if
   respond_to?(:issues_url)
```

Comment:
If you require chef 12 you can nuke these respond_to and issues_url and source_url.